### ISSUE TITLE
NTR: N-acetyl-aspartyl-glutamate absent in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8478]
New term request N-acetyl-aspartyl-glutamate absent in CSF (segal_170306192935): NAAG; cerebrospinal fluid  UMLS: 4314740

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000765
- label: Decreased CSF N-acetyl-aspartyl-glutamate concentration
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: 
- def: The concentration of N-acetyl-aspartyl-glutamate (NAAG) in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: N-acetyl-aspartyl-glutamate (NAAG) is the most abundant dipeptide in the brain, where it acts as a neuromodulator of glutamatergic synapses by activating presynaptic metabotropic glutamate receptor 3 (mGluR3).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8478
- synonyms:
